pom.xml

```diff
@@ -522,9 +480,27 @@
                 <version>${mockito.version}</version>
                 <scope>test</scope>
             </dependency>
+
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>${jaxb-api.version}</version>
+                <scope>provided</scope>
+            </dependency>
+
+            <dependency>
```

Comment:
This dependency has exclusively (and not intentionally) been used by the JAX-RS plugin which is no longer part of the core distribution, can be removed.

Reply:
Removed, commited and pushed.